doc/source/api.rst

```diff
@@ -500,6 +500,7 @@ Miscellaneous
    :toctree: _generated/
 
    aslarray
+   from_frame
```

Comment:
needs to add from_series too

Reply:
The current implementation is partially. I prefer not.